src/DIRAC/WorkloadManagementSystem/Service/JobStateUpdateHandler.py

```diff
@@ -211,6 +211,8 @@ def _setJobStatusBulk(cls, jobID, statusDict, force=False):
                 log.debug("Set job end time", endTime)
 
         # We should only update the status to the last one if its time stamp is more recent than the last update
+        attrNames = []
```

Comment:
then you should remove these no ? 
https://github.com/DIRACGrid/DIRAC/blob/4607f579faa7b52fc0c25643116f182750d29066/src/DIRAC/WorkloadManagementSystem/Service/JobStateUpdateHandler.py#L249-L250

Reply:
OK done.